services/core-web/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -115,8 +116,7 @@ export class MineTailingsInfo extends Component {
       value: record.engineer_of_record?.party_guid,
       label: record.engineer_of_record?.party.name,
     };
-    const itrb = !isNil(record.has_itrb) ? record.has_itrb.toString() : record.has_itrb;
-    const newRecord = { ...record, has_itrb: itrb };
+    const newRecord = { ...record };
 
```

Comment:
We no longer have to spread `...record` in a new const since we aren't manipulating anything. You can pass `record` directly.

Reply:
Fixed.